openedx/core/djangoapps/content/course_overviews/models.py

```diff
@@ -207,15 +257,18 @@ def load_from_module_store(cls, course_id):
         with store.bulk_operations(course_id):
             course = store.get_course(course_id)
             if isinstance(course, CourseDescriptor):
-                course_overview = cls._create_from_course(course)
+                course_overview = cls._create_or_update_from_course(course)
                 try:
                     with transaction.atomic():
                         course_overview.save()
+                        # Remove and recreate all the course tabs - when the course overview
+                        # had the tabs which are not present in course descriptor anymore.
```

Comment:
It's unclear what the 2nd phrase of this comment means.  Can you explain further?  Or, remove that phrase if it doesn't add any value?